sympy/integrals/laplace.py

```diff
@@ -1314,7 +1342,8 @@ def _inverse_laplace_rational(fn, s, t, plane, simplify):
     return result, And(*conditions)
 
 
-def _inverse_laplace_transform(fn, s_, t_, plane, simplify=True, dorational=True):
+def _inverse_laplace_transform(fn, s_, t_, plane, simplify=True,
+                               dorational=True):
```

Comment:
```suggestion
def _inverse_laplace_transform(
    fn, s_, t_, plane, simplify=True, dorational=True):
```
I think that it's better to use [hanging indents](https://peps.python.org/pep-0008/#indentation) for these sort of things because after long names, because you recognize that everything pushes to the right too much when you align with the opening delimiter